pages/common/xml-select.md

```diff
@@ -0,0 +1,25 @@
+# xml select
+
+> Select from XML documents using XPATHs.
+> Tip: use `xml elements` to display the XPATHs of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Select all elements matching "XPATH1" and print the value of the element "XPATH2":
```

Comment:
Is XPATH2 some kind of sub-element of XPATH1? Otherwise I don't understand what this does.

Reply:
Yes. Manual reading really is required to learn xpaths. The match is a for-all selection of all xpath1. Within those, it prints xpath2. Example: for all users, print the user's name.

Reply:
Wouldn't it make it at least a bit easier if we write `sub-element`?

Reply:
- Select all elements matching "XPATH1" and print the value of their sub-element "XPATH2":